src/coreclr/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -543,7 +553,7 @@ private static bool FunctionJustThrows(MethodIL ilBody)
         partial void DetermineIfCompilationShouldBeRetried(ref CompilationResult result)
         {
             // If any il bodies need to be recomputed, force recompilation
-            if (_ilBodiesNeeded != null)
+            if ((_ilBodiesNeeded != null) || InfiniteCompileStress.Enabled)
```

Comment:
Does this mean that with <code>InfiniteCompileStress</code> Crossgen2 literally never finishes? Would it be more general to use some such variable to define the number of compilation iterations so that we could subsequently create a pipeline with the option set to something like 100 (that might even roughly mimic the GC stress slowdown)?

Reply:
That sounds interesting. But of course, that wouldn't be "infinite" stress. We're already in a state where seeing a real problem takes days.